### ISSUE TITLE
Disabled MaskedTextBox - Hard to read on Apple devices

### ISSUE THREAD

**Issue by RaptorCZ** [telerik/kendo-ui-core#5282]
I found, that disabled `MaskedTextBox` is almost unreadable on `Apple` devices.

To test you can use Telerik page https://demos.telerik.com/kendo-ui/maskedtextbox/mvvm

Select Bootstrap 3 Theme and uncheck Enable button.  
Text in input is almost invisible.

Or use this simple fiddle https://codepen.io/raptor/pen/OJLBpOL

**Comment by dmanova** [comment]
Indeed, some of the themes has light gray font color when a component is disabled and in iOS they become even lighter. That makes them hard to read.

You could try some other color themes, until we fix that issue. Or change the style for example:
`<style>
.k-state-disabled {
    opacity: .8;
}
</style>`
or
`<style>
.k-input, .k-multiselect-wrap, .k-textbox>input, input.k-textbox {
    color: black;
}
</style>`
